### PR TITLE
Close  #115: Allow user to store non-openPMD information

### LINKED ISSUE #115: Allow user to store non-openPMD information
Today I had a request from a prospective user of openPMD: can the user store other datasets in the HDF5-openPMD files, which are **not meant** to be read by the openPMD parser / viewer ? (for instance because they don't fall either in the category mesh or or in the category particle)

From my point of view, the best way to do this is to store this data outside of the `basePath` (in this case the parser will not try to read it). 

@ax3l : Does that make sense ? If yes, I think it would be good to add it as a side-remark in the standard.

### PR DESCRIPTION
**Pull request by RemiLehe**
Close #115: This allows user to store additional data, outside of the basePath.

### CODE REVIEW

**Comment by ax3l** [comment]
thx for the PR! :)

File: STANDARD.md [hunk 1]
```diff
@@ -100,6 +100,16 @@ Each file's *root* directory (path `/`) must at leat contain the attributes:
                    of `%T` with the integer value of the iteration, e.g.,
                    `/data/%T` becomes `/data/100`
     - allowed value: fixed to `/data/%T/` for this version of the standard
+	- remark: all the data that is formatted according to the present
```
Comment (ax3l) [inline STANDARD.md]: pls no tab :)
can you use "note: " instead? :)

File: STANDARD.md [hunk 2]
```diff
@@ -100,6 +100,16 @@ Each file's *root* directory (path `/`) must at leat contain the attributes:
...
+      standard (i.e. both the meshes and the particles) is to be
+      stored within a path of the form given by `basePath` (e.g. in
+      the above example, the data will be stored within the path `/data/100/`).
+      If, for various reasons, a user wants to store **additional
+      data** that is not (or cannot be) formatted according to the
+      present standard (e.g. fields on an unstructured mesh),
+      this can be done be storing this data within a path that **is not**
+      of the form given by `basePath` (e.g. `/extra_data`). In this
```
Comment (ax3l) [inline STANDARD.md:111]: we could write more simple: "within a path that does **not reside** inside `basePath` (e.g. `/extra_data`)."

File: STANDARD.md [hunk 3]
```diff
@@ -100,6 +100,16 @@ Each file's *root* directory (path `/`) must at leat contain the attributes:
                    of `%T` with the integer value of the iteration, e.g.,
                    `/data/%T` becomes `/data/100`
     - allowed value: fixed to `/data/%T/` for this version of the standard
+	- remark: all the data that is formatted according to the present
+      standard (i.e. both the meshes and the particles) is to be
+      stored within a path of the form given by `basePath` (e.g. in
+      the above example, the data will be stored within the path `/data/100/`).
+      If, for various reasons, a user wants to store **additional
```
Comment (ax3l) [inline STANDARD.md]: formatting-wise: I think we currently _stress_ important facts that do not introduce keywords, etc. only with a simple italic `*stressed fact*` in the text.

**Comment by RemiLehe** [comment]
Thanks for the comments! I updated the PR.

**Pull request merged**